.github/workflows/pr.yaml

```
@@ -55,3 +57,19 @@ jobs:
       arch: "amd64"
       container_image: "rapidsai/ci:latest"
       run_script: "ci/build_docs.sh"
+  wheel-build:
+    needs: checks
+    secrets: inherit
+    uses: rapidsai/shared-action-workflows/.github/workflows/wheels-pure-build.yml@cuda-120
+    with:
+      build_type: pull-request
+      package-name: cuxfilter
+      package-dir: python
```

Let's fix this one too, since it's the one being tested in PRs. :)
```suggestion
      package-dir: python/
```